### PR TITLE
Fix `Uncaught TypeError: _this.refs.scrollView.scrollTo is not a function`

### PR DESCRIPTION
**Pull request by flyskywhy**
when using https://github.com/flyskywhy/react-web

ref to https://facebook.github.io/react/docs/refs-and-the-dom.html

### CODE REVIEW

**Comment by arribbar** [comment]
@flyskywhy Can you describe more what you intend to with this PR ?
I'll add some comments directly in your code as well.

File: src/index.js [hunk 1]
```diff
@@ -432,7 +432,7 @@ export default class extends Component {
     })
 
     // trigger onScrollEnd manually in android
-    if (!animated || Platform.OS === 'android') {
+    if (!animated || Platform.OS !== 'ios') {
```
Comment (arribbar) [inline src/index.js]: why changing this line ?

File: src/index.js [hunk 2]
```diff
@@ -419,10 +419,10 @@ export default class extends Component {
     if (state.dir === 'x') x = diff * state.width
     if (state.dir === 'y') y = diff * state.height
 
-    if (Platform.OS === 'android') {
-      this.refs.scrollView && this.refs.scrollView[animated ? 'setPage' : 'setPageWithoutAnimation'](diff)
+    if (Platform.OS !== 'ios') {
```
Comment (arribbar) [inline src/index.js]: Why changing this line ?

**Comment by flyskywhy** [comment]
The reason of changing `Platform.OS === 'android'` to `Platform.OS !== 'ios'`, is to match the logic of `Platform.OS === 'ios'` in renderScrollView(), then this react-native-swiper can be used with not just ios and android, but also windows and web described in `exports.platforms = ['ios', 'android', 'windows', 'web'];` of [https://github.com/facebook/react-native/blob/0.44-stable/packager/defaults.js](https://github.com/facebook/react-native/blob/0.44-stable/packager/defaults.js), for example, `Platform.OS === 'web'` in [https://github.com/flyskywhy/react-web](https://github.com/flyskywhy/react-web) :stuck_out_tongue:

**Pull request merged**